udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -441,6 +458,13 @@ function setGranularity(value: string) {
 function submit() {
   validateRequiredRules().then(valid => {
     if(valid) {
+      if (toValue(userOrganization) === "user") {
```

Comment:
Do you think it's better to add a const with `"user"` or a function to do the check ?